apps/web/vibes/soul/primitives/blog-post-card/index.tsx

```diff
@@ -2,66 +2,86 @@ import { clsx } from 'clsx';
 import Image from 'next/image';
 import Link from 'next/link';
 
-export interface BlogPostCardBlogPost {
-  id: string;
-  author?: string | null;
-  content: string;
-  date: string;
-  image?: {
-    src: string;
-    alt: string;
+/**
+ * This component supports various CSS variables for theming. Here's a comprehensive list, along
+ * with their default values:
+ *
+ * ```css
+ * :root {
+ *   --blog-post-card-focus: hsl(var(--primary));
+ *   --blog-post-card-primary-text: hsl(var(--foreground));
+ *   --blog-post-card-secondary-text: hsl(var(--contrast-400));
+ *   --blog-post-card-tertiary-text: hsl(var(--foreground)/15%);
+ *   --blog-post-card-image-background: hsl(var(--contrast-100));
+ *   --blog-post-card-font-family: var(--font-family-body);
+ *   --blog-post-card-summary-text: hsl(var(--contrast-400));
+ *   --blog-post-card-author-date-text: hsl(var(--foreground));
+ * }
+ * ```
+ */
+export function BlogPostCard({
+  blogPost,
+  className,
+}: {
+  blogPost: {
```

Comment:
I still feel like this component shouldn't have a `blogPost` prop as it implies we're making a decision on what a "blog post" entity should look like. The properties of `blogPost` should be at the top level props, IMO